### PR TITLE
Feature/biotype

### PR DESCRIPTION
**Pull request by tgrego**
This pull request modifies the production schema, updating the existing biotype table to the master tables.
A new column 'so_acc' that represents the Sequence Ontology accession of the biotypes is added and populated.
The changes are mostly in patch_92_93_b.sql, which follows the patch_92_93_a.sql from pull request #95.

Doing this will allow a new biotype table in the core schema, using the existing infrastructure of the master tables to copy over the data.

ensembl-orm is also updated to make use of the schema changes.

### CODE REVIEW

**Comment by thomasmaurel** [comment]
Thanks for that @tgrego. Travis seems to be reporting a conflict, could you please have a look?

**Comment by tgrego** [comment]
This should now pass all checks as soon as pull request #2 on ensembl-orm gets merged.

**Comment by thomasmaurel** [comment]
Thanks @tgrego but I don't know who looks after ensembl-orm. I will wait for the ensembl-orm pull request to be merged first

**Comment by nerdstrike** [comment]
It's managed by Web

File: modules/Bio/EnsEMBL/Production/Pipeline/Common/Base.pm [hunk 1]
```diff
@@ -739,4 +739,15 @@ sub has_genes {
   return $count;
 }
 
+sub get_biotype_group {
+  my ($self, $group) = @_;
+
+  my $ba = Bio::EnsEMBL::Registry->get_adaptor($self->param('species'), 'core', 'Biotype');
+
+  my $biotypes = $ba->fetch_all_by_group_object_db_type( $group, 'gene', 'core' );
```
Comment (magaliruffier) [inline modules/Bio/EnsEMBL/Production/Pipeline/Common/Base.pm]: this method is not available in the BiotypeAdaptor

Comment (tgrego) [reply modules/Bio/EnsEMBL/Production/Pipeline/Common/Base.pm]: The implementation is pending merge of https://github.com/Ensembl/ensembl/pull/230

Comment (nerdstrike) [reply modules/Bio/EnsEMBL/Production/Pipeline/Common/Base.pm]: Unless I am mistaken, this should be ready now.

Comment (thomasmaurel) [reply modules/Bio/EnsEMBL/Production/Pipeline/Common/Base.pm]: yep, merging now

**Pull request merged**